### PR TITLE
Fully adpots previously ported bottle sprite and fixes reagent overlays

### PR DESCRIPTION
**Pull request by Bm0n**
## What Does This PR Do

Changes almost all bottles to the previously ported bottle new bottle sprite.

Removes the old r4407 bottles

Fixes bottle overlaying. Reagent overlays now actually appear on bottle sprites.

Adjusts beaker and bottle reagent overlaying to intervals of 20u. Previously an overlay state was not being used and overlays only changed in intervals of 25u, the modern sprites were made with intervals of 20u in mind.

Standardizes bottles to a volume of 50u. Note: This does not increase the amount of chemicals in a bottle, only the size of the container is increased.

Replaces the "reagent bottle" sprite with a new one based on the modern bottle.

## Why It's Good For The Game

We ported a new bottle sprite not long ago but it isn't used that much and its overlays were completely broken.

After fixing the overlays I have chosen to switch all of the chemical bottles over to the new bottle sprite because it is in the 3/4 style and a lot better looking than the ancient **_r4407_** bottles that we are currently using alongside the new bottle.

A unified look for bottles is much better than the chaotic one we have right now. It is as if someone was blindfolded and threw darts at a board to figure out which chemical gets what bottle sprite, it doesn't look good IMO and the old sprites are heavily dated.

I decided to keep the "reagent bottle" around and respite it to match the new bottle but with an amber hue and a darker reagent overlay. Reagent bottles are the bottles that spawn in reagent closets, they are simply fluff.  Pictures of this sprite can be found below.

Finally, setting the generic bottle size to a volume of 50u makes the most sense to me and it works better with the fixed overlays. Right now you can produce an infinite amount of 50u sized bottles at any chem master, it makes no sense that bottles bought at a medical vendor are 30u in size despite sharing the same sprite and name as those produced at chem masters.

Do keep in mind this is not changing the amount of chemicals you get per bottle at a medical vendor, only the maximum amount of chemicals a bottle can hold is changed

## Images of changes

pr1

pr2

## Testing

I spawned myself in and tested every reagent overlay state for each bottle and beaker

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>

## Changelog

:cl: Bmon
add: Changed most chemical bottle sprites over to the recently added version.
imageadd: Added a new sprite for reagent bottles.
imagedel: Removed old r4407 bottles.
tweak: All bottles now have a generic size of 50u.
fix: Bottle and beaker overlaying fixed.
/:cl:

### CODE REVIEW

**Comment by CinnamonSnowball** [comment]
Is there any particular reason to remove all the previous bottle sprites as opposed to respriting them? As is, this PR *really* hurts variety, removing 3 unique bottle sprites and replacing one of them with just a recolor of the only remaining one.

**Comment by Bm0n** [comment]
> Is there any particular reason to remove all the previous bottle sprites as opposed to respriting them? As is, this PR _really_ hurts variety, removing 3 unique bottle sprites and replacing another one with just an ugly recolor of the only remaining base bottle sprite.

They're old flat 2d sprites from r4407 that work with underlays as opposed to overlays that probably should have been removed when we got the new bottle sprite. As it stands people people seems to lean towards using the new bottle sprite even though the overlays for it are broke in the master branch.

File: code/modules/reagents/reagent_containers/glass_containers.dm [hunk 1]
```diff
@@ -142,23 +142,21 @@
 /obj/item/reagent_containers/glass/beaker/update_overlays()
 	. = ..()
 	if(reagents.total_volume)
-		var/image/filling = image('icons/obj/reagentfillings.dmi', src, "[icon_state]10")
+		var/image/filling = image('icons/obj/reagentfillings.dmi', src, "[icon_state]1")
```
Comment (lewcc) [inline code/modules/reagents/reagent_containers/glass_containers.dm:145]: Is there no zero/empty icon state?

File: code/modules/reagents/reagent_containers/chemical_bottle.dm [hunk 2]
```diff
@@ -5,37 +5,38 @@
 	name = "bottle"
 	desc = "A small bottle."
 	icon = 'icons/obj/chemical.dmi'
-	icon_state = "round_bottle"
+	icon_state = "bottle"
 	item_state = "atoxinbottle"
 	amount_per_transfer_from_this = 10
-	possible_transfer_amounts = list(5,10,15,25,30)
+	possible_transfer_amounts = list(5,10,15,25,30,40,50)
```
Comment (Burzah) [inline code/modules/reagents/reagent_containers/chemical_bottle.dm:11]:
```suggestion
	possible_transfer_amounts = list(5, 10, 15, 25, 30, 40, 50)
```

**Review by Burzah — APPROVED** [review]

**Review by DGamerL — APPROVED** [review]

**Pull request merged**